### DISCUSSION TITLE
If a video download fails, will podsync eventually try to repair it?

### DISCUSSION THREAD

**Discussion by distbit0** [mxpv/podsync#309] [General]
I often run into videos failing to download when adding channels to my subscriptions, and was wondering whether these will just never be added or if they will be revisited by podsync later on and attempt to be re-downloaded.

Thank you

**Comment by mxpv** [comment]
Yes, once downloaded Podsync sets episode's status to `Downloaded`.
If the download was interrupted, it'll stuck in pending state and will be retried next time.
